account_move_base_import/models/account_journal.py

```diff
@@ -0,0 +1,287 @@
+# -*- coding: utf-8 -*-
+# © 2011 Akretion
+# © 2011-2016 Camptocamp SA
+# © 2013 Savoir-faire Linux
+# © 2014 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+import sys
+import traceback
+import os
+from openerp import _, api, fields, models
+from ..parser.parser import new_move_parser
+from openerp.exceptions import UserError, ValidationError
+from operator import attrgetter
+
+
+class AccountJournal(models.Model):
+    _name = 'account.journal'
+    _inherit = ['account.journal', 'mail.thread']
+
+    used_for_import = fields.Boolean(
+        string="Journal used for import")
+
+    commission_account_id = fields.Many2one(
+        comodel_name='account.account',
+        string='Commission account')
+
+    import_type = fields.Selection(
+        [('generic_csvxls_so', 'Generic .csv/.xls based on SO Name')],
+        string='Type of import',
+        default='generic_csvxls_so',
+        required=True,
+        help="Choose here the method by which you want to import bank "
+        "statement for this profile.")
```

Comment:
We should not speak about bank statement or profile anymore.
A lot of description are still the old ones. I don't think it should block the PR, but we should fix it. (I can do it once it is merged if needed)